qbo_face_detector:
  ros__parameters:
    # face_classifier_path: "/home/qbo-v2/qbo_ws/src/qbo_vision/config/classifier/haarcascade_frontalface_default.xml"
    face_classifier_path: "/home/qbo-v2/qbo_ws/src/qbo_vision/config/classifier/haarcascade_frontalface_alt2.xml"
    # alternative_face_classifier_path: "/home/qbo-v2/qbo_ws/src/qbo_vision/config/classifier/haarcascade_frontalface_alt2.xml"
    lbp_classifier_path: "/home/qbo-v2/qbo_ws/src/qbo_vision/config/classifier/lbpcascade_frontalface.xml"

    check_Haar: 1
    check_track_obj: 1
    undetected_threshold: 5
    distance_threshold: 1.2
    send_to_face_recognizer: true
    print_recognized_face: true
